app/src/main/java/com/github/sdp/ratemyepfl/activity/MainActivity.kt

```diff
@@ -40,10 +57,66 @@ class MainActivity : AppCompatActivity() {
         )
 
         setupActionBarWithNavController(navController, appBarConfiguration)
+
+        startLocationService()
     }
 
     override fun onSupportNavigateUp(): Boolean {
         return navController.navigateUp(appBarConfiguration)
     }
+
+    /**
+     * Starts the Location Service and binds this Activity as a Listener
+     */
+    private fun startLocationService() {
```

Comment:
We can maybe unify this location service with what Nicolas did in the Map activity. A task for next sprint !